packages/opentelemetry-exporter-collector/test/browser/CollectorMetricExporter.test.ts

```diff
@@ -166,25 +162,12 @@ describe('CollectorMetricExporter - web', () => {
       });
 
       it('should log the error message', done => {
-        const spyLoggerError = sinon.spy();
-        const handler = loggingErrorHandler({
-          debug: sinon.fake(),
-          info: sinon.fake(),
-          warn: sinon.fake(),
-          error: spyLoggerError,
-        });
-        setGlobalErrorHandler(handler);
-        const spyLoggerDebug = sinon.stub(collectorExporter.logger, 'debug');
         spyBeacon.restore();
         spyBeacon = sinon.stub(window.navigator, 'sendBeacon').returns(false);
 
-        collectorExporter.export(metrics, () => {});
-
-        setTimeout(() => {
-          const response: any = spyLoggerError.args[0][0] as string;
-          assert.ok(response.includes('sendBeacon - cannot send'));
-          assert.strictEqual(spyLoggerDebug.args.length, 1);
-
+        collectorExporter.export(metrics, result => {
+          assert.deepStrictEqual(result.code, ExportResultCode.FAILED);
+          assert.ok(result.error?.message.includes('cannot send'));
```

Comment:
the message is still `sendBeacon - cannot send` so why shorter check then ?

Reply:
i just rewrote the code without thinking about what was in place, i will add it back